quarkus/set-quarkus-version.sh

```diff
@@ -48,24 +52,32 @@ fi
 
 QUARKUS_BOM=$(curl -f -s "$QUARKUS_BOM_URL")
 
-echo "Setting Quarkus version: $QUARKUS_VERSION"
-$(mvn versions:set-property -f ../pom.xml -Dproperty=quarkus.version,quarkus.build.version -DnewVersion="$QUARKUS_VERSION" 1> /dev/null)
+if [ "$QUARKUS_VERSION" != "nightly" ]; then
+    echo "Setting Quarkus version: $QUARKUS_VERSION"
+    $(mvn versions:set-property -f ../pom.xml -Dproperty=quarkus.version,quarkus.build.version -DnewVersion="$QUARKUS_VERSION" 1> /dev/null)
+fi
```

Comment:
I was wondering if we really can't use it in nightly. It's essentially what we do here:
https://github.com/keycloak/keycloak/blob/ecbd856176fa81203be0fe07464299d39883e9f8/.github/scripts/prepare-quarkus-next.sh#L53-L54
Shouldn't we have a single place to set the versions?

Reply:
@vmuzikar in short, if 999-SNPSHOT is passed, the script sets quarkus.version to a wrong version (999.0.0-SNPSHOT or something similar), which makes pom.xml unresolvable and the mvn versions:set-property then doesn't work.

I can provide the details once I get to the computer

Reply:
Sorry, I was not clear. Wanted to suggest we could perhaps run the script without any params [here](https://github.com/keycloak/keycloak/blob/bb52f950374b34dbd4c42edae3dfdb2465b8be52/.github/scripts/prepare-quarkus-next.sh#L53). This will resolve to the correct version [here](https://github.com/keycloak/keycloak/blob/bb52f950374b34dbd4c42edae3dfdb2465b8be52/quarkus/set-quarkus-version.sh#L35) and correct branch [here](https://github.com/keycloak/keycloak/blob/bb52f950374b34dbd4c42edae3dfdb2465b8be52/quarkus/set-quarkus-version.sh#L36). Then we'll have a single place to change Quarkus versions instead of a script and a GH Action.

Reply:
Addressed.